build.gradle

```diff
@@ -345,18 +345,18 @@ project('spring-rabbit-junit') {
 	dependencies { // no spring-amqp dependencies allowed
 
 		compile "org.springframework:spring-core:$springVersion"
-		compile "junit:junit:$junit4Version"
+		compile ("junit:junit:$junit4Version", optional)
```

Comment:
I'm not sure if you have fixed all the tests into JUnit 5 style, but I guess we need to think eventually about removing `testCompile ("junit:junit:$junit4Version") {` to disallow to use it accidentally in tests

Reply:
Right - but we still have a long way to go before we can do that 😄